docs/detections/alert-suppression.asciidoc

```diff
@@ -9,17 +9,29 @@ Alert suppression requires a https://www.elastic.co/pricing[Platinum or higher s
 preview::[]
 --
 
-Alert suppression allows you to reduce the number of repeated or duplicate detection alerts created by a <<create-custom-rule,custom query detection rule>> each time the rule runs. 
+Alert suppression allows you to reduce the number of repeated or duplicate detection alerts created by a <<create-custom-rule,custom query detection rule>>.
 
-Normally, when a rule matches multiple source events, multiple alerts are created, one for each event. When alert suppression is configured, matching events are grouped by a specified field, and only one alert is created for each group per rule execution. You can also specify multiple fields to group events by unique combinations of values.
+Normally, when a rule matches multiple source events, it creates multiple alerts, one for each event. When alert suppression is configured, matching events are grouped by a specified field, and only one alert is created for each group. You can configure alert suppression to create alerts each time the rule runs, or once within a specified time window. You can also specify multiple fields to group events by unique combinations of values.
 
 The {security-app} displays several indicators in the Alerts table and the alert details flyout when a detection alert is created with alert suppression enabled. You can view the original events associated with suppressed alerts by investigating the alert in Timeline.
 
 NOTE: Alert suppression is not available for Elastic prebuilt rules. However, if you want to suppress alerts for a prebuilt rule, you can duplicate it, then configure alert suppression on the duplicated rule.
 
 === Configure alert suppression
 
-You can configure alert suppression when you create or edit a <<create-custom-rule,custom query rule>>. When configuring the rule type (the *Define rule* step for a new rule, the *Definition* tab for an existing rule), enter one or more field names in *Suppress Alerts By*. Configure other rule settings, then save and enable the rule.
+You can configure alert suppression when you create or edit a custom query rule. Refer to <<create-custom-rule>> for detailed instructions.
+
+. When configuring the rule type (the *Define rule* step for a new rule, the *Definition* tab for an existing rule), enter one or more field names in *Suppress alerts by*. 
+. Select how often to create alerts for duplicate events:
++
+--
+* *Per rule execution*: Create an alert each time the rule runs and matches duplicate events.
+* *Per time period*: Create one alert for all matching duplicate events within a specified time window, beginning when the rule first matches an event and creates the alert.
++
+For example, if a rule runs every 5 minutes but you don't need alerts that frequently, you can set the suppression time period to a longer time, such as 1 hour. If the rule matches an event, it creates an alert at that time, and for the next hour any subsequent matches will be suppressed.
+--
+
```

Comment:
An image of the alert suppression window w/ the options would be good here, IMO.

Reply:
Added in https://github.com/elastic/security-docs/pull/3026/commits/60c37d4ae9e4489755b10bcdb7224c8821b02e94